src/amber/cli/templates/model.cr

```diff
@@ -21,7 +22,7 @@ module Amber::CLI
     end
 
     def table_name
-      @table_name ||= "#{@name}s"
+      @table_name ||= "#{Inflector.pluralize(@name)}"
```

Comment:
I know this was already merged, but I think a refactor would be helpful. The code `Inflector.pluralize(@name)` shows up in lots of places. Why not have this in the template:
```
def plural_name
  @plural_name ||= Inflector.pluralize(@name)
end
```
Then `plural_name` can be used everywhere, and if amber ever swaps this inflector for another one (like what @faustinoaq mentioned) then it just has to change in this one place.

Also, the inflector being called that many times might incur some unnecessary overhead.

Reply:
I agree :+1: